docs/sdk/typescript.mdx

```diff
@@ -45,23 +42,42 @@ const keypair = Keypair.fromMnemonicSet(mnemonic)[0]
 ## Use the SDK with the Keypair
 
 ### Get the balance
+Use a publicKey to check your Kin balance.
 
 ```ts
 const balance = await sdk.balance(keypair.publicKey)
 ```
 
 ### Request an Airdrop
+Send Kin to a publicKey on the Solana devnet.
+
+Airdrops take some time to complete as we make sure the transaction has been confirmed on the blockchain before returning a response. 
 
 ```ts
 const signature = await sdk.requestAirdrop(keypair.publicKey, "42000")
 ```
 
 ### Make a Transfer
+Send Kin from a Keypair (as you need the privateKey) to a publicKey on the blockchain.
+
+You can optionally set the commitment level before returning ('Confirmed', 'Finalized' or Processed), the transaction type, or add a referenceId / referenceType. 
 
 ```ts
 const result = await sdk.makeTransfer({
-  amount: "42",
+  	amount: "42",
 	destination: "ALisrzsaVqciCxy8r6g7MUrPoRo3CpGxPhwBbZzqZ9bA",
 	owner: keypair,
 })
 ```
+Here's the makeTransfer Type for reference: 
+```ts
+makeTransfer({ amount, commitment, destination, owner, referenceId, referenceType, type, }: {
+    amount: string;
+    commitment?: Commitment;
+    destination: string;
+    owner: Keypair;
+    referenceId?: string;
+    referenceType?: string;
+    type?: TransactionType;
+}): Promise<import("../generated").AppTransaction>;
```

Comment:
I think here we can remove the 'import' part, and return `Promise<AppTransaction>` instead.